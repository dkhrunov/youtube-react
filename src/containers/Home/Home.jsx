import React, { Fragment } from 'react';
import SideBar from '../SideBar/SideBar';
import VideoGrid from '../../components/VideoGrid/VideoGrid';
import './Home.scss';

const Home = (props) => {
	return (
		<Fragment>
			<SideBar />
			<div className='home'>
				<div className='responsive-video-grid-container'>
					<VideoGrid title='Trending' />
					<VideoGrid title='Autos & Vencles' hideDivider={true} />
				</div>
			</div>
		</Fragment>
	);
};

export default Home;
