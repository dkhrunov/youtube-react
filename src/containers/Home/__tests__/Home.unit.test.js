import React from 'react';
import { shallow } from 'enzyme';
import Home from '../Home';

describe('Home', () => {
	test('renders without props', () => {
		const wrapper = shallow(<Home />);
		expect(wrapper).toMatchSnapshot();
	});
});
