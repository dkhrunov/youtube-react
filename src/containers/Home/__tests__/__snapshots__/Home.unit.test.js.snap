// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders without props 1`] = `
<Fragment>
  <SideBar />
  <div
    className="home"
  >
    <div
      className="responsive-video-grid-container"
    >
      <VideoGrid
        title="Trending"
      />
      <VideoGrid
        hideDivider={true}
        title="Autos & Vencles"
      />
    </div>
  </div>
</Fragment>
`;
